src/apps/vpn/telemetry/metrics.yaml

```diff
@@ -150,3 +150,20 @@ session:
     notification_emails:
       - mcleinman@mozilla.com
     expires: never
+
+  connection_health_stable_count:
+    type: counter
+    lifetime: ping
+    send_in_pings:
+      - vpnsession
+    description: |
+      Count of times that the connection health check is successful
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/VPN-4700
+    data_reviews:
+      - TO BE ADDED
```

Comment:
```suggestion
      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/6783
```

Reply:
🤦🏻  Thanks, good catch. Updated this to the review that was done.